optimizer_wrapper.rb

```diff
@@ -501,6 +504,39 @@ def self.job_remove(api_key, id)
 
   def self.check_result_consistency(expected_value, results)
     [results].flatten(1).each{ |result|
+      if result[:routes].any?{ |route| route[:activities].any?{ |a| a[:waiting_time].to_i < 0 } }
+        log 'Computed waiting times are invalid', level: :warn
+        raise RuntimeError, 'Computed waiting times are invalid' if ENV['APP_ENV'] != 'production'
```

Comment:
Do we want to raise an error ? This might return no result because of a wrong waiting time. Shouldn't we rather raise an error to debug internally instead of blocking optimization ? 

I can understand that we do not want to return a wrong value, but at the same time we would have no bypass for user to get its result. 

In order to minimize error in return value we could eventually replace negative values by zero and keep computing ? (while sending a warning to sentry)

Reply:
This raise won't happen in production environment. It will only raise during development, the behavior is similar to the raise performed for `total_time`